doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -596,6 +596,9 @@ curl localhost:1026/v2/entities/Room1/attrs/temperature/value -s -S -H 'Content-
 Finally, the `PUT /v2/entities/{id}/attrs` operation can be used to replace all the attributes
 of a given entity, i.e. removing previously existing ones.
 
+You can also update entity with forcedUpdate URI option for discarding previous value and update 
```

Comment:
I'd suggest not including this text here. Note this section is about "Update entity" and notifications (where forcedUpdate has its effect) has not been introduced yet to the reader.